installer/InstallBuilder/linuxrpm.py

```diff
@@ -146,9 +147,14 @@ def StageAndProperlyNameRPM(self):
                 self.fullversion_dashed + '.rhel.' + \
                 str(self.variables['PFMAJOR'])  + '.' + self.variables['PFARCH'] + '.rpm'
         elif self.variables['PFDISTRO'] == 'ULINUX':
```

Comment:
Removed the PPC check in linuxrpm.py and made the DISTRO as REDHAT in https://github.com/Microsoft/SCXcore/pull/59/commits/7088e66dcaa14b36a14ffb3c24f502d1c5f4ea23